### ISSUE TITLE
请问下vue您是如果调用扩展的js从而去执行扩展中的js的方法的呢

### ISSUE THREAD

**Issue by jesty666** [miru-project/repo#311]
### Extension information

1

### Steps to reproduce

可以给个思路吗或者原理

### Miru version

1

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

**Comment by lattaai3** [comment]
Hello, I tried to fix the issue

This is what I did:

To improve the robustness of the extension, we'll add error handling and retry logic to the search function in the yymanhua.com.js file. This will help handle potential network issues or temporary failures when searching for manga.

I also created pull request: https://github.com/miru-project/repo/pull/312


> [!CAUTION]
> Disclaimer: This fix was created by Latta AI and you should never merge before you check the correctness of generated code!
>
> The fix provided by Latta AI might not be complete and it can serve as an inspiration.

---
This bug was fixed for free by Latta AI - https://latta.ai/ourmission

If you no longer want Latta AI to attempt fixing issues on your repository, you can block this account.

**Comment by hualiong** [comment]
你是啥意思嘞？如果是Vue的写的页面一般都是前后端分离架构，你只需要找到它的后端服务API调用即可

**Comment by jesty666** [comment]
> 你是啥意思嘞？如果是Vue的写的页面一般都是前后端分离架构，你只需要找到它的后端服务API调用即可

他不是有扩展插件js吗 之前作者开源了vue版本 我想知道他是怎么做到这个扩展插件放到前端来获取数据

**Comment by hualiong** [comment]
哦，这我就不知道了🤷‍♀️
